src/utils.ts

```diff
@@ -25,19 +25,24 @@ export const generateClientInfo = () => ({
   },
 });
 
-export const containsUnsafeChars = (s) => {
+export const containsUnsafeChars = (s: string) => {
```

Comment:
You can delete this file and just use `jose.base64url.encode(JSON.stringify(...)` in place of `jsonToBase64`

Reply:
I didnt delete the file, but I removed most of the util functions and uses jose for base64.